libs/feature/search/src/lib/router/constants.ts

```diff
@@ -0,0 +1,6 @@
+import { Component } from '@angular/core'
+
+export const ROUTER_STATE_KEY = 'router'
+
+export class SearchRouteComponent extends Component {}
+export class MetadataRouteComponent extends Component {}
```

Comment:
not very comfortable with this but it seems mandatory.

Reply:
I know, I feel like we're misusing the router somehow, but in our case I think using `router-outlet` isn't the best approach. And these components are harmless and not exported outside of the module.